packages/backend/server/src/plugins/copilot/controller.ts

```diff
@@ -129,7 +136,6 @@ export class CopilotController {
     @CurrentUser() user: CurrentUser,
     @Req() req: Request,
     @Param('sessionId') sessionId: string,
-    @Query('messageId') messageId: string,
```

Comment:
is this a breaking change? Maybe older client will not work any longer with it. Shall we make it compatible for old client at least for one major version?

Reply:
message id will get in `@Query() params: Record<string, string | string[]>`, no behavier change